src/Notifaika/Core.hs

```diff
@@ -61,7 +61,6 @@ repostUpdates = do
         let (cachedEvents', newEvents) =
                 detectNewEvents (cachedEvents, currentEvents)
         forM_ newEvents $ \Event{message} ->
-            -- TODO move withRoom above forM_
```

Comment:
It's not possible to implement in an easy way, in order to do that we need to make Gitter monad to be an `Eff`, so we could interchange effects. Theoretically it's possible to implement entirely on Notifaika level, and it may be an interesting task, though was not sure that it worth that. I can put TODO back, though.